larcorealg/Geometry/GeometryCore.cxx

```diff
@@ -1353,6 +1353,16 @@ namespace geo {
 
   } // GeometryCore::IntersectSegments()
 
+
+  //......................................................................
+  geo::Point_t GeometryCore::WiresIntersect
+    (geo::WireGeo const& wire1, geo::WireGeo const& wire2) const
+  {
+    assert(!wire2.isParallelTo(wire1));
+    return geo::WiresIntersection(wire1, wire2);
+  } // GeometryCore::WiresIntersect()
+
+
   //......................................................................
   bool GeometryCore::WireIDsIntersect(
```

Comment:
Many of `GeometryCore`'s member functions appear to be merely "convenience" functions that do not actually use `GeometryCore`'s internal state (such as `WireIDsIntersect`).  This creates unnecessary entanglements.  The `GeometryCore` class should gradually be pruned of its "add-ons", so that only member functions that require access to data members are provided.

Tackling that problem was not in the scope of this PR, but it motivates the reason for not supporting a `geo::WiresIntersection` member function.